### PR TITLE
doc: Improve readability of section in High-Level API tutorial

### PR DESCRIPTION
**Pull request by qwepoizt**
This is just a minor change to improve readability. After having read through all items of the PR guidelines, I believe none of them applies.

### CODE REVIEW

File: doc/tutorials/highlevel.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ default = 1.1
 ```
 
 In Elektra a specification is defined through the metadata of keys in the `spec` namespace. The specification above contains metadata for
-three keys the parent key (`@`), `@/mydouble` and `@/myfloatarray/#`. The `#` at the end of `myfloatarray/#` indicates that it is an array.
+three keys: 1. the parent key (`@`), 2. `@/mydouble` and 3. `@/myfloatarray/#`. The `#` at the end of `myfloatarray/#` indicates that it is an array.
```
Comment (markus2330) [inline doc/tutorials/highlevel.md:40]: Maybe you can even make a proper markdown enumeration?

Please do not forget to add release notes and to make the build server happy.

**Comment by markus2330** [comment]
Thank you very much for creating the PR!

> I believe none of them applies.

Adding something to the release notes always applies because every contribution needs to be honored properly.

File: doc/tutorials/highlevel.md (hunk 1 above)
Comment (qwepoizt) [reply doc/tutorials/highlevel.md]: Thanks for the suggestion, I converted it into an enum and added release notes!

**Comment by qwepoizt** [comment]
@markus2330 1 check failed because Cirrus CI reports a broken link (http://www.complang.tuwien.ac.at/raab/akademy.pdf) in doc/news/2018-08-18_0.8.24.md.

Without `www` the link does not timeout but sends a 404: http://complang.tuwien.ac.at/raab/akademy.pdf

The broken link seems to be unrelated to my changes, I'm not sure how to proceed?

**Comment by kodebach** [comment]
Since there is now a merge conflict, you should rebase the PR branch onto master. You can ignore the broken link. We can merge the PR, if everything else works. 

Maybe we can fix the link (it is a tuwien.ac.at link after all), but if not we can also add the link to [the whitelist](https://github.com/ElektraInitiative/libelektra/blob/master/tests/linkchecker.whitelist).

**Comment by mpranj** [comment]
I suggest ignoring the broken link for now. As announced in #160 our build infrastructure is down. The broken link is affected by the same planned outage which should be resolved today. I would advise against merging during this period.

The merge conflict will need to be resolved by rebasing on current master as @kodebach already said. Otherwise it looks good to me. :+1:

**Comment by markus2330** [comment]
Yes, please add www, this was a recent change in the complang infrastructure. complang.tuwien.ac.at does not redirect to www.complang.tuwien.ac.at anymore.

**Comment by qwepoizt** [comment]
Thank you all for the input, I will attend to it in the coming days.

**Comment by qwepoizt** [comment]
I rebased my patch-1 and all checks have passed. Please, feel free to merge this PR!

File: doc/news/_preparation_next_release.md [hunk 2]
```diff
@@ -214,6 +214,8 @@ _(@tucek)_
 - Added a short Visual Studio 2019 tutorial (/doc/tutorials/contributing-windows.md) _(Dominic Jäger)_
 - Added hint regarding WSL filesystem configuration (/doc/tutorials/contributing-windows.md) _(@tucek)_
 - Fixed broken link in yanlr-plugin readme _(@lawli3t)_
+- Minor readability improvement in [](/doc/tutorials/highlevel.md) _(Tobias Schubert @qwepoizt)_
+- Minor readability improvement in [highlevel.md](/doc/tutorials/highlevel.md) _(Tobias Schubert @qwepoizt)_
```
Comment (mpranj) [inline doc/news/_preparation_next_release.md:218]:
```suggestion
- Minor readability improvement in [highlevel.md](/doc/tutorials/highlevel.md) _(Tobias Schubert @qwepoizt)_
```

**Comment by mpranj** [comment]
Thank you so much for the docu improvements!

_(The last build error is a known sporadic error #3679)_

**Pull request merged**